libs/vue/src/schematics/component/schematic.ts

```diff
@@ -0,0 +1,90 @@
+import { strings } from '@angular-devkit/core';
+import {
+  apply,
+  applyTemplates,
+  chain,
+  mergeWith,
+  move,
+  Rule,
+  SchematicContext,
+  Tree,
+  url,
+} from '@angular-devkit/schematics';
+import {
+  formatFiles,
+  getProjectConfig,
+  offsetFromRoot,
+  projectRootDir,
+  ProjectType,
+  toClassName,
+  toFileName,
+} from '@nrwl/workspace';
+import { ComponentSchematicSchema } from './schema';
+
+interface NormalizedSchema extends ComponentSchematicSchema {
+  componentName: string;
+  projectRoot: string;
+  componentPath: string;
+  styleExtension: string;
+  destinationPath: string;
+}
+
+function normalizeOptions(
+  host,
+  options: ComponentSchematicSchema,
+  context: SchematicContext
+): NormalizedSchema {
+  const componentName = toClassName(options.name);
+
+  const componentPath = options.directory
+    ? `${toFileName(options.directory)}`
+    : '';
+
+  const projectName = options.project;
+  const projectConfig = getProjectConfig(host, projectName);
+
+  const projectRoot = `${projectRootDir(projectConfig.projectType)}/${
+    options.project
+  }`;
+
+  // depending on projectType build destination path of component
+  const destinationPath =
+    projectConfig.projectType === ProjectType.Application
+      ? `${projectRoot}/src/components/${componentPath}`
+      : `${projectRoot}/src/lib/${componentPath}`;
+
+  const styleExtension = options.style ?? 'css';
+
+  return {
+    ...options,
+    componentName,
+    projectRoot,
+    componentPath,
+    styleExtension,
+    destinationPath,
+  };
+}
+
+function createComponent(options: NormalizedSchema): Rule {
+  return mergeWith(
+    apply(url(`./files`), [
+      applyTemplates({
+        ...options,
+        ...strings,
+        toClassName,
+        offsetFromRoot: offsetFromRoot(options.destinationPath),
+      }),
+      move(options.destinationPath),
```

Comment:
With the changes to `normalizeOptions` this needs to be changed to `move(options.componentPath)`.